script/Arcueid/Day 12/12_00_ARC12_2B.txt

```diff
@@ -253,17 +253,17 @@ Is that... Mario's voice coming out of a speaker...?
 [sha:93cc4b8f616bb8b362bcf2a2c390de776e2922b8]{
 -- Page 93, Offset 30802.
 -- 「“この街に派遣された司祭代行だよ。
-"I'm the Acting Presbyter deployed in this city.
+"The Acting Presbyter sent to this city.
 }
 [sha:09a148d1f9df3b23fe49f59000d3f161637ad380]{
 -- Page 93, Offset 30803.
 -- 名前はマーリオゥ・ジャッロ・ベスティーノ。
-The name's Mario Gallo Bestino.
+Name's Mario Gallo Bestino.
 }
 [sha:98449e4a8c8425c439134b1ed26a76c5866b49b6]{
 -- Page 93, Offset 30804.
 -- テメェならこれで意味が通るだろ、ヘビ野郎”」
-A guy like you should know what that means, you god damn snake."
+Surely you know what entails, you snake fuck."
```

Comment:
LMAO this one is SPOT-ON sopranos dialogue. BELLISSIMO

Reply:
what that entails